### PR TITLE
created past project link, rmoved cards from coaching and advice

### PR DESCRIPTION
**Pull request by omartehsin1**
# Summary | Résumé

Created a past projects link to move all the cards from the coaching and advice page to the new link.

### CODE REVIEW

File: content/en/past-projects/_index.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+type: section
+layout: past-projects
+title: Past Projects
+alias: [/past-projects-fr/]
```
Comment (omartehsin1) [inline content/en/past-projects/_index.html:5]: Update to FR translation

File: content/fr/past-projects/_index.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+type: section
+layout: past-projects
+title: Past Projects FR
+aliases: [/past-projects/]
+url: /past-projects-fr/
```
Comment (omartehsin1) [inline content/fr/past-projects/_index.html:6]: Update both title and URL to FR translation

File: i18n/fr.yaml [hunk 3]
```diff
@@ -222,6 +224,8 @@ roadmap-link:
   other: "https://numerique.canada.ca/feuille-de-route-2025/"
 how-cds-started-link:
   other: "https://numerique.canada.ca/les-debuts-du-snc/"
+past-projects-link:
+  other: "/past-projects-fr/"
```
Comment (omartehsin1) [inline i18n/fr.yaml:228]: Update to FR translation

File: i18n/fr.yaml [hunk 4]
```diff
@@ -212,6 +212,8 @@ roadmap:
   other: "Feuille de route vers 2025"
 how-cds-started:
   other: "Les débuts du SNC"
+past-projects:
+  other: "(FR) Past Projects"
```
Comment (omartehsin1) [inline i18n/fr.yaml:216]: Update to FR translation

**Review by victoriachanimal — APPROVED** [review]

**Comment by victoriachanimal** [comment]
Thanks @omartehsin1 !!! LGTM!

**Pull request merged**